worlds/sc2/Items.py

```diff
@@ -68,175 +213,807 @@ def get_full_item_list():
+    ItemNames.HELS_ANGEL: ItemData(505 + SC2WOL_ITEM_ID_OFFSET, "Mercenary", 5, SC2Race.TERRAN,
+                                   classification=ItemClassification.progression,
+                                   description="Mercenary Vikings"),
+    ItemNames.DUSK_WINGS: ItemData(506 + SC2WOL_ITEM_ID_OFFSET, "Mercenary", 6, SC2Race.TERRAN,
+                                   description="Mercenary Banshees"),
+    ItemNames.JACKSONS_REVENGE: ItemData(507 + SC2WOL_ITEM_ID_OFFSET, "Mercenary", 7, SC2Race.TERRAN,
+                                         description="Mercenary Battlecruiser"),
+
+    ItemNames.ULTRA_CAPACITORS: ItemData(600 + SC2WOL_ITEM_ID_OFFSET, "Laboratory", 0, SC2Race.TERRAN,
+                                         description="""
+                                         Increases attack speed of all Terran units with each weapon upgrade.
+                                         """),
+    ItemNames.VANADIUM_PLATING: ItemData(601 + SC2WOL_ITEM_ID_OFFSET, "Laboratory", 1, SC2Race.TERRAN,
+                                         description="""
+                                         Increases life of all Teran units with each armor upgrade.
+                                         """),
+    ItemNames.ORBITAL_DEPOTS: ItemData(602 + SC2WOL_ITEM_ID_OFFSET, "Laboratory", 2, SC2Race.TERRAN,
+                                       description="""
+                                       Supply depots are built instantly.
+                                       """),
+    ItemNames.MICRO_FILTERING: ItemData(603 + SC2WOL_ITEM_ID_OFFSET, "Laboratory", 3, SC2Race.TERRAN,
+                                        description="""
+                                        Increases Vespene Gas harvesting speed.
+                                        """),
+    ItemNames.AUTOMATED_REFINERY: ItemData(604 + SC2WOL_ITEM_ID_OFFSET, "Laboratory", 4, SC2Race.TERRAN,
+                                           description="""
+                                           Refineries don't require harvesters to operate
+                                           """),
+    ItemNames.COMMAND_CENTER_REACTOR: ItemData(605 + SC2WOL_ITEM_ID_OFFSET, "Laboratory", 5, SC2Race.TERRAN,
+                                               description="""
+                                               Command Centers can train two SCVs at once.
+                                               """),
+    ItemNames.RAVEN: ItemData(606 + SC2WOL_ITEM_ID_OFFSET, "Laboratory", 6, SC2Race.TERRAN,
+                              description="""
+                              Aerial Support unit.
+                              """),
+    ItemNames.SCIENCE_VESSEL: ItemData(607 + SC2WOL_ITEM_ID_OFFSET, "Laboratory", 7, SC2Race.TERRAN,
+                                       classification=ItemClassification.progression,
+                                       description="""
+                                       Aerial Support unit. Can repair mechanical units.
```

Comment:
I recommend describing Ravens and Science Vessels as "Caster units" if you're not going to include abilities. That or there needs to be a way to tell the player "support" = "caster"